### ISSUE TITLE
How to add a Icon to fontawesome? 

### ISSUE THREAD

**Issue by pqekypk2** [MichaelBarney/LinkFree#95]
Im working with SunMoon template, but i don't know how to add tiktok icon to the font 

anyone can help me?

**Comment by Noevenzahn** [comment]
Hey, that should be easy, check out the docs here: 
https://fontawesome.com/icons/tiktok?s=brands

FontAwesome icons can be changed with the class names for the `<i>` tag. 
A TikTok logo would look like this:
`<i class="fa-brands fa-tiktok"></i>`

**Comment by pqekypk2** [comment]
no, i´m already tryit but this template use a exported font in a attached file, so the brands not inlcuded dont work

**Comment by Noevenzahn** [comment]
Than it seems like you have to create an own file/kit and add it to the website. Refer to this guide:
https://fontawesome.com/docs/web/

**Comment by pqekypk2** [comment]
Thanks i will try it.

**Comment by pqekypk2** [comment]
i remove the font file and change the import statement
